### ISSUE TITLE
NameError: global name 'GotoWarning' is not defined

### ISSUE THREAD

**Issue by Arxcis** [technocake/goto#82]
```terminal
jonasjso in ~/git/github/technocake/goto on develop
$ goto goto

Ah hoy!

    - The Magic word goto does not exist.

    Are you sure it exists in the project you are in now?
    Type:

        goto list

    to see all shortcuts in this project.
    
Traceback (most recent call last):
  File "/usr/local/bin/the_real_goto.py", line 11, in <module>
    load_entry_point('magicgoto', 'console_scripts', 'the_real_goto.py')()
  File "/Users/jonasjso/git/github/technocake/goto/goto/the_real_goto.py", line 33, in main
    output, err = run_command(magic, command, args)
  File "/Users/jonasjso/git/github/technocake/goto/goto/the_real_goto.py", line 79, in run_command
    return commands.default(magic, command)
  File "/Users/jonasjso/git/github/technocake/goto/goto/commands/default.py", line 14, in default
    return None, GotoWarning('magicword_does_not_exist', magicword=magicword)
NameError: global name 'GotoWarning' is not defined
```

**Comment by Arxcis** [comment]
fixed by #83

**Comment by technocake** [comment]
I see another related issue here -- 

goto <non-existing>  prints the warning message twice..

[![asciicast](https://asciinema.org/a/fEyP2vgPooqRF2eb4gy6NVYBI.svg)](https://asciinema.org/a/fEyP2vgPooqRF2eb4gy6NVYBI)
